src/libwrapper.hpp

```diff
@@ -28,7 +28,7 @@ class Library : public Libs {
 	Library(bool uinput, bool uoutput, bool colorize_output):
 		utf8_input_(uinput), utf8_output_(uoutput), colorize_output_(colorize_output) {}
 
-	bool process_phrase(const char *loc_str, IReadLine &io, bool force = false);
+	bool process_phrase(const char *loc_str, IReadLine &io, bool force = false, bool use_json = false);
 private:
 	bool utf8_input_;
     bool utf8_output_;
```

Comment:
why not make `use_json` class member of `Library`, like `utf8_input_` and `utf8_output_`?